lib/flame/dispatcher.rb

```diff
@@ -155,7 +156,7 @@ def cached_tilts
 		## Return response if HTTP-method is OPTIONS
 		def try_options
 			return unless request.http_method == :OPTIONS
-			allow = @app_class.router.routes.dig(request.path)&.allow
+			allow = @endpoint&.allow
```

Comment:
Write tests for case with `OPTIONS` on route with optional parameters and without their values in path.

Before that PR it would not work, as I understand it, but now it will be.

Reply:
works as you expected. Done